src/analyses/static_analysis.h

```diff
@@ -86,6 +86,9 @@ class domain_baset
   // this computes the join between "this" and "b"
   // return true if "this" has changed
 
+  bool has_been_seen() const { return seen; }
```

Comment:
It isn't clear to me what `seen` means, and since it is now being added to the interface I would appreciate if these functions have doxy-comments explaining what they mean.